### PR TITLE
Added DBUIQuit command

### PR DESCRIPTION
**Pull request by bgfxc4**
Added the command, because I want to be able to close the DBUI when opening nvim-tree to only have one sidebar opened.

### CODE REVIEW

File: plugin/db_ui.vim [hunk 1]
```diff
@@ -115,6 +115,7 @@ augroup END
 
 command! DBUI call db_ui#open('<mods>')
 command! DBUIToggle call db_ui#toggle()
+command! DBUIQuit call db_ui#quit()
```
Comment (kristijanhusak) [inline plugin/db_ui.vim:118]: Lets call it `DBUIClose` and the function `db_ui#close()`, just to have it properly as an opposite of `open`.

**Review by kristijanhusak — CHANGES_REQUESTED** [review]

**Review by kristijanhusak — APPROVED** [review]

**Comment by kristijanhusak** [comment]
Thanks!

**Comment by ueaner** [comment]
> Added the command, because I want to be able to close the DBUI when opening nvim-tree to only have one sidebar opened.

Looking forward to your works.
I have a question, do I need to check if `dbui` is open?

**Comment by kristijanhusak** [comment]
> > Added the command, because I want to be able to close the DBUI when opening nvim-tree to only have one sidebar opened.
> 
> Looking forward to your works. I have a question, do I need to check if `dbui` is open?

There's no need for that.

**Comment by ueaner** [comment]
> > do I need to check if `dbui` is open?
> 
> There's no need for that.

Thank you for your reply.
I made an implementation based on the open state of dbui.

https://github.com/ueaner/nvimrc/blob/0c75033f981f4e171c6b87550bc186bffaaef6e7/lua/utils.lua#L76-L119

Am I missing something?
Can you share your approach?

**Pull request merged**